vmaas/reposcan/database/repository_store.py

```diff
@@ -72,13 +72,13 @@ def _import_certificate(self, cert_name, ca_cert, cert, key):
             key = None
         cur = self.conn.cursor()
         try:
-            cur.execute("select id from certificate where name = %s", (cert_name,))
+            cur.execute("select id, ca_cert, cert, key from certificate where name = %s", (cert_name,))
             cert_id = cur.fetchone()
```

Comment:
This is no longer just a `cert_id` but seem like the whole row.

Reply:
Fixed